draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -379,17 +383,17 @@
 	  identifying the key.
 	</t>
 	<t>
-	  The following example (using JSON notation) demonstrates such a declaration
+	  The following example (using CBOR diagnostic notation) demonstrates such a declaration
 	  in the CWT Claims Set of a CWT:
 	</t>
 	<figure>
 	  <artwork><![CDATA[
   {
-   "iss": "https://server.example.com",
-   "aud": "https://client.example.org",
-   "exp": 1361398824,
-   "cnf":{
-     "kid": "dfd1aa97-6d8d-4575-a0fe-34b96de2bfad"
+   /iss/ 1 : "coaps://server.example.com",
+   /aud/ 3 : "coaps://client.example.org",
+   /exp/ 4 : 1361398824,
+   /cnf/ 8 : {
+     /kid/ 2 : b64'dfd1aa97-6d8d-4575-a0fe-34b96de2bfad'
```

Comment:
It's not clear to me that we want to use the base64-encoding of an ASCII GUID as a COSE key ID.  Maybe change this to a 16-byte binary value (which could be 0xdfd1aa976d8d4575a0fe34b96de2bfad).